indico/MaKaC/webinterface/pages/static.py

```diff
@@ -48,6 +48,9 @@ def _getMenu(self):
 
 class WPTPLStaticConferenceDisplay(WPStaticEventBase, WPTPLConferenceDisplay):
 
+    def getJSFiles(self):
```

Comment:
Does this fix it on all pages of a conference that use custom JS that's not in the base module? Otherwise it might make more sense to simply remove `getJSFile` from `WPStaticEventBase`